webpack.config.js

```diff
@@ -107,10 +133,22 @@ const config = {
 			{
 				test: /\.jsx?$/,
 				enforce: 'post',
-				use: {
-					loader: 'babel-loader',
-					options: babelSettings,
-				},
+				use: [
+					{
+						loader: 'thread-loader',
+						options: {
+							workers: Math.max( 2, Math.floor( os.cpus().length / 2 ) ),
```

Comment:
This is what makes this baby fly now, try it in one of those new fancy 6-core Macbook Pros for maximum awesomeness :)